### DISCUSSION TITLE
hotkey for project panel & other settings

### DISCUSSION THREAD

**Discussion by yanballas** [zed-industries/zed#18751] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

hi guys! I have a two questions.
1. Is there any way I can set a hotkey for toggle project panel? i want to use ctrl+shift or something else.
2. how i can turn off tips for writing code? I attached a screenshot. 
thank you!
![Снимок экрана от 2024-10-04 23-14-39](https://github.com/user-attachments/assets/8bd2e132-7d50-47b7-8fc7-f65e619fcf21)

**Comment by SomeoneToIgnore** [comment]
For 2., `"show_signature_help_after_edits": false,` could be added to the user settings.

For 1., you can hover the button which shows the current hotkey:
![image](https://github.com/user-attachments/assets/1a0b42f3-dfc6-490d-84a0-f3b17314bb94)

right clicking the panel button allows to move certain panels around, which may change the keybindings.
"moving around" is called docking, there are left/right/bottom ones with its own keybindings
https://zed.dev/docs/key-bindings might help with the customization of those.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
